kbase-extension/static/kbase/js/widgets/narrative_core/kbaseNarrativeSideImportTab.js

```diff
@@ -285,13 +284,13 @@ define (
             }
             var numberOfTabs = importMethodIds.length;
             if (numberOfTabs > 1) {
-                var $header = $('<div>');
-                var $body = $('<div>');
+                $header = $('<div>');
+                $body = $('<div>');
                 this.widgetPanelCard2.append($header).append($body);
             }
 
-            for (var methodPos in importMethodIds) {
```

Comment:
That's mainly a style thing imposed by following eslint. Since methodPos was already declared in the loop on line 279, it doesn't need to be declared again.

I agree it should still be forced to keep to local scope, but the linter was pretty angry about it, and I'm exploring using those, based on @sychan 's suggestion.